app/templates/services/edit_submission_section.html

```diff
@@ -35,36 +35,28 @@
 
 {% block save_button %}
 
-  {% if next_section_name and not force_return_to_summary %}
+  {% if not next_question or force_return_to_summary %}
     {%
       with
-      label="Save and continue",
-      name = "continue_to_next_section",
+      label="Save and return to service overview",
```

Comment:
I think we should drop the word "service" from the button text, to make it more generic for whatever kind of thing you're adding - having the word "Service" on the button would be confusing during the flow for , e.g. labs or specialists.  (An alternative would be to pull the right noun from `lot.data.unitSingular`, but that feels like overkill to me.)